x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form_advanced_options.tsx

```diff
@@ -139,6 +172,17 @@ export const RuleFormAdvancedOptions = (props: RuleFormAdvancedOptionsProps) =>
     onFlappingChange,
   } = props;
 
+  const {
+    application: {
+      capabilities: {
+        rulesSettings: { writeFlappingSettingsUI },
```

Comment:
This needs to be mocked in `x-pack/plugins/monitoring/public/alerts/alert_form.test.tsx`

Reply:
nice catch!